suite-native/app/.env.develop

```diff
@@ -3,4 +3,5 @@ COMMIT_HASH=
 CHANGELOG=
 VERSION=
 BUILD_NUMBER=
+CODESIGN_BUILD=true
```

Comment:
I would expect `develop` and maybe even `staging` environment to have `CODESIGN_BUILD=false`, how else could QA look at a new message-system config before releasing it to production?

Reply:
Good spot! 😬 That is an error left from testing. fixed in 97296bcbe0